src/api/goods.ts

```diff
@@ -3,17 +3,30 @@ import { Good } from '../types/Good';
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
+export function getData<T>(): Promise<T> {
```

Comment:
In the getData function, you are not specifying what type of data you are expecting as a response. This can potentially lead to type errors. Consider replacing `<T>` with `<Good[]>`.